manim/utils/config_utils.py

```diff
@@ -314,6 +324,27 @@ def _parse_cli(arg_list, input=True):
         "--config_file", help="Specify the configuration file",
     )
 
+    parser.add_argument(
+        "-v", "--verbose",
+        type=str,
+        help="Verbosity level. Also changes the ffmpeg log level unless the latter is specified in the config",
+        metavar="loglevel",
+        choices=constants.VERBOSE_CHOICES,
+    )
+
+    def _str2bool(s):
+        if s == "True":
+            return True
+        elif s == "False":
+            return False
+        else:
+            raise argparse.ArgumentTypeError("True or False expected")
+    parser.add_argument(
+        "--progress_bar",
+        type=_str2bool,
```

Comment:
Is this necessary? I'm pretty sure that argument parsers support booleans by default.

Reply:
I thought so but it is not the case. I think argparse uses the given type as a function to change the string to the desired type. The thing is `bool("False")` actually gives `True`. In fact, any non empty string passed to `bool` gives `True`.
I checked this [discussion](https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse). One suggested solution is to either give an implemented function changing a string to a boolean (with adequate mapping) or to add two flags, one being the same name as the first one but starting by `--no-` In this case it would be one flag `--progress_bar` and the other `--no-progress_bar`. I chose to implement a simple conversion function to keep the CLI flag the same as the config file in which one would put `progress_bar=True`.